boot/freeldr/freeldr/include/ver.h

```diff
@@ -21,9 +21,9 @@
 
 /* Just some stuff */
 #define VERSION         "FreeLoader v3.0"
-#define COPYRIGHT       "Copyright (C) 1998-" COPYRIGHT_YEAR " ReactOS Team"
+#define COPYRIGHT       "Copyright (C) 1998-" COPYRIGHT_YEAR " ReactOS Project"
```

Comment:
Do you want to use this opportunity and change the first copyright year to 1996 consistently? 
It's the year that is prominently advertised on our boot screen.

Reply:
I think I can change this as well, but maybe FreeLoader first appeared in 1998 (didn't checked that yet).

Reply:
yes, freeldr didn't appear on day one in ReactOS.

Reply:
Ok so it actually appeared in 2001 in revision afdf923c65707f267bf3088e9f7712f3d8d869cb (SVN r1904), so it will be okay to change this first copyright year.